corso_website_fe/src/Main Page Components/BannerMain.css

```diff
@@ -71,4 +71,9 @@
     line-height: normal;
     text-transform: uppercase;
     background: none;
-}
\ No newline at end of file
+}
+
+.row-banner{
```

Comment:
What does this do ?

Reply:
stops a sliding bar from appearing at the bottom